wordpress-mobile/WordPress-iOS
# Stats: add style guide extension for stats specific styles

Fixes #n/a

To test:
- Go to Stats with:
  - A post that has views, likes, or comments.
  - A post that has no views, likes, or comments.
  - A site that has no posts.

Verify the styles have not changed. You can reference the styles in Zeplin or the screenshots on https://github.com/wordpress-mobile/WordPress-iOS/pull/10390.
